### ISSUE TITLE
[Enhancement] Remove Duplicates

### ISSUE THREAD

**Issue by meichthys** [LycheeOrg/Lychee#1762]
A Deduplication or "Remove Duplicates" feature would be welcomed!

**Comment by ildyria** [comment]
Finding the duplicates is the easy part. Providing the interaction to know which one to remove is another piece of cake in the other hand...

That being said, I will not be working on this enhancement (other mights or project for volunteers).

**Comment by meichthys** [comment]
Prompting the user to choose which duplicate to remove should be fine.

**Comment by ildyria** [comment]
> Prompting the user to choose which duplicate to remove should be fine.

Like I said, easier said than done. :sweat_smile:

**Comment by meichthys** [comment]
Agreed. Just filed this issue for tracking here: https://github.com/meichthys/foss_photo_libraries#free-and-opensource-photo-libraries

**Comment by ildyria** [comment]
I did not know about this repo, however this is a bit incorrect:
- [Photo Discovery](https://github.com/meichthys/foss_photo_libraries/blob/main/features.md#photo-discovery) is covered with the _On This Day_ smart album.
- [Photo Map](https://github.com/meichthys/foss_photo_libraries/blob/main/features.md#photo-map) is covered: we do have the map in the top right corner.
:)

**Comment by meichthys** [comment]
Awesome. Thanks! I'll get that updated 👍

**Comment by eip618** [comment]
is there a function when *uploading* new images that catches duplicates?

**Comment by ildyria** [comment]
> is there a function when _uploading_ new images that catches duplicates?

A bit late, but Yes.
And there is a settings which allows you to decide what happens in such cases.
